scopes/component/code/code.section.tsx

```diff
@@ -12,6 +12,7 @@ export class CodeSection implements Section {
   navigationLink = {
     href: '~code',
     children: <MenuWidgetIcon icon="Code" tooltipContent="Code" />,
+    simpleText: 'Code',
```

Comment:
text to be used in the header dropdown (instead of showing the icon like we do in desktop res)